modules/apps/site-initializer/site-initializer-liferay-learn/src/main/resources/site-initializer/fragments/group/public-sites-navigation/fragments/public-sites-navigation/index.html

```diff
@@ -31,10 +31,35 @@
 					</div>
 				</div>
 				[#if configuration.showLanguageMenu]
-					<div class="align-self-center language p-2">
-						<span class="globe-icon pr-1 opacity-50">[@clay["icon"] symbol="globe-lines"/]</span>
-						<span class="font-weight-semi-bold language-text opacity-80">EN</span>
-						<span class="language-dropdown-icon opacity-30">[@clay["icon"] symbol="caret-bottom-l"/]</span>
+					<div class="language-selector-container">
+						<div class="font-size-paragraph-xsmall language language-selector p-2 utility-nav-link">
```

Comment:
@liaoethan  The font sizes in the language dropdown don't seem to match what's in figma (16px in figma vs 14px) which makes the font dropdown not aligned with the sites menu: 

![image](https://user-images.githubusercontent.com/16581685/206568085-7d008f0e-29a2-4ff7-bbd2-f481611fbcca.png)

I noticed you added `font-size-paragraph-xsmall` here but that doesn't seem to have any effect- can you take a look?

Reply:
@liaoethan  Please review the changes I made here: https://github.com/brianchandotcom/liferay-portal/pull/127490

If you could also test that and let me know if anything is broken, that'd be great :-) It looks good to me when I tested.

I found that we have some CSS that is never used so please try to be careful about that.

Reply:
Thanks @allen-ziegenfus! I'll be more careful about the unused CSS in the future.